src/main/java/frc/robot/subsystems/intakerelease/IntakeRelease.java

```diff
@@ -100,7 +102,11 @@ public void holdCone() {
   }
 
   public void holdCube() {
-    state = State.HOLD_CUBE;
+    if (inputs.cubeLimitSwitch) {
```

Comment:
This will only get run when calling it from a command.  Instead, could you put this logic in `periodic()`?

```
HOLD_CUBE -> intakeReleaseIO.setPercent(inputs.cubeLimitSwitch ? -0.1 : -0.35);
```